src/scripting/natives/natives_cutil.yaml

```diff
@@ -0,0 +1 @@
+GET_STRING_FROM_SYMBOL_LARGE: pPointer:pointer -> string
```

Comment:
instead of `pPointer` it should be `pointer`

Reply:
Addressed!